templates/extensions/typescript/integration-test.ts

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+import { expect } from "chai";
+
+describe("greet-the-world", () => {
+  it("should respond with the configured greeting", async () => {
+    const expected = "Hello World from greet-the-world";
+
+    const httpFunctionUri = "http://localhost:5001/demo-test/us-central1/ext-greet-the-world-greetTheWorld/";
```

Comment:
This assumes functions emulator always runs on port 5001, maybe we can make that a commandline option?

Reply:
We hard code that port in the firebase.json we write (see integration-test.json). There might be a better way to pipe it through from the emulator as an environment variable (and then rely on the default port discovery mechanism instead of hard coding it), but this should work for now.

In the case when 5001 is taken, this will error out with a clear message, too.